operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -71,6 +71,10 @@ type CentralSpec struct {
 	// Monitoring configuration.
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=9,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced"}
 	Monitoring *GlobalMonitoring `json:"monitoring,omitempty"`
+
+	// System configuration.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=10,xDescription={"urn:alm:descriptor:com.tectonic.ui:advanced"}
+	System *GlobalSystemSpec `json:"system,omitempty"`
```

Comment:
nit: I wonder if we should have this separate System group.

It would have been nice to have this as a separate group that applies to all components (and then have all global configuration in it), but now that we already global settings (e.g. Egress, TLS, ImagePullSecrets, Monitoring etc)  in `CentralSpec` directly I'm not sure if it makes sense to create a new group just for this one setting.